### ISSUE TITLE
Spread resources are labelled multiple times

### ISSUE THREAD

**Issue by mkaito** [Pandemoneus/Resource-Labels#2]
It seems resources, especially around the borders, are more fuzzy in 0.16. This appears to cause multiple resource labels to be placed upon a single patch.

![rl1](https://user-images.githubusercontent.com/20434/34047843-828652b2-e1a9-11e7-8425-08924f090270.png)

![rl3](https://user-images.githubusercontent.com/20434/34047932-cad7e2e2-e1a9-11e7-845b-7b7524fd9079.png)

The same happens for oil patches, which are very frequently labelled multiple times.

![rl2](https://user-images.githubusercontent.com/20434/34047871-9eb603d8-e1a9-11e7-9af8-d464ab74cbe9.png)

**Comment by Pandemoneus** [comment]
Yes, I am aware that this is the case. A label is created for each resource patch, and in a resource patch all resource entites must be connected. If there is a gap between two resource patches, they are counted seperately. I will take a look at how I can fix this and add it as configurable behaviour.
